server.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"api-booking-time/config"
+	"api-booking-time/infrastructure/datastore"
+)
+
+var (
+	db *datastore.DbRepository
+)
+
+func main() {
+	config.ReadConfig()
+	db = datastore.OpenDb()
+
+	r := mux.NewRouter()
+	api := r.PathPrefix("/api/v1").Subrouter()
+	api.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "api booking-time")
+	})
+	api.HandleFunc("/centres", getCentres).Methods(http.MethodGet)
```

Comment:
I think this logic should be moved to a different function, and maybe in another package.

Maybe inside `router/centre.go` for example.